app/ts/components/simulationExplaining/customExplainers/EnsEventExplainer.tsx

```diff
@@ -0,0 +1,434 @@
+import { RpcNetwork } from '../../../types/rpc.js'
+import { RenameAddressCallBack } from '../../../types/user-interface-types.js'
+import { EnsEvent } from '../../../types/visualizer-types.js'
+import { dataStringWith0xStart } from '../../../utils/bigint.js'
+import { assertNever } from '../../../utils/typescript.js'
+import { SmallAddress } from '../../subcomponents/address.js'
+import { Ether } from '../../subcomponents/coins.js'
+import { EditEnsNamedHashCallBack, EnsNamedHashComponent } from '../../subcomponents/ens.js'
+
+type EnsEvenExplainerParam = {
+	ensEvent: EnsEvent,
+	textColor: string,
+	renameAddressCallBack: RenameAddressCallBack,
+	editEnsNamedHashCallBack: EditEnsNamedHashCallBack,
+	rpcNetwork: RpcNetwork,
+}
+
+const VisualizeEnsEvent = ({ ensEvent, textColor, editEnsNamedHashCallBack, renameAddressCallBack, rpcNetwork }: EnsEvenExplainerParam) => {
+	const textStyle = `color: ${ textColor }; margin-bottom: 0px; display: inline-block`
+	switch(ensEvent.subType) {
+		case 'ENSAddrChanged': return <div class = 'log-table-4'>
+			<div class = 'log-cell'>
+				<p class = 'ellipsis paragraph' style = { textStyle }>
+					Change
+				</p>
+			</div>
+			<div class = 'log-cell'>
+				<EnsNamedHashComponent type = { 'nameHash' } nameHash = { ensEvent.logInformation.node.nameHash } name = { ensEvent.logInformation.node.name } editEnsNamedHashCallBack = { editEnsNamedHashCallBack }/>
```

Comment:
```suggestion
				<EnsNamedHashComponent type = 'nameHash' nameHash = { ensEvent.logInformation.node.nameHash } name = { ensEvent.logInformation.node.name } editEnsNamedHashCallBack = { editEnsNamedHashCallBack }/>
```

Note: There are several occurrences of these parameter that only consist of string that can be directly passed without wrapping it in parens